src/vdaf.rs

```diff
@@ -290,6 +328,22 @@ impl<F: FieldElement> AggregateShare<F> {
     }
 }
 
+impl<F: FieldElement> Encode for AggregateShare<F> {
+    fn encode(&self, bytes: &mut Vec<u8>) {
+        encode_items_u16(bytes, &self.0);
+    }
+}
+
+impl<F: FieldElement> Decode<()> for AggregateShare<F> {
+    type Error = VdafError;
+
+    fn decode(_decoding_parameter: &(), bytes: &mut Cursor<&[u8]>) -> Result<Self, Self::Error> {
+        let items = decode_items_u16(&(), bytes)?;
```

Comment:
Done! The tricky thing here is that we now have to define the associated type on `trait Vdaf` as:
```
    /// An Aggregator's share of the aggregate result.
    type AggregateShare: Aggregatable<OutputShare = Self::OutputShare> + Decode<usize> + Encode;
```
Note `Decode<usize>`. Someday a VDAF may come along whose `AggregateShare` has a decoding parameter that isn't `usize`, and then we'd have to do something like add `type AggregateShareDecodingParameter` to `trait Vdaf`, but I think `usize` will get us pretty far, because it'll work for anything whose output share is a vector of field elements.